packages/test/stochastic-test-utils/src/describeFuzz.ts

```diff
@@ -0,0 +1,55 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+function createFuzzSuite(
+    tests: (this: Mocha.Suite, testCount: number) => void,
+    testCount: number,
+) {
+    return function(this: Mocha.Suite) {
+        tests.bind(this)(testCount);
+    };
+}
+
+export type DescribeFuzzSuite = (
+    name: string,
+    tests: (
+        this: Mocha.Suite,
+        /**
+         * The number of tests this suite should produce, to within a constant factor.
+         * It's up to the suite author to decide which parameters to vary in order to scale up the number of tests.
+         */
+        testCount: number,
+    ) => void)
+=> Mocha.Suite | void;
+
+export type DescribeFuzz = DescribeFuzzSuite & Record<"skip" | "only", DescribeFuzzSuite>;
+
+export interface FuzzDescribeOptions {
+    defaultTestCount?: number;
+}
+
+export const defaultOptions: Required<FuzzDescribeOptions> = {
+    defaultTestCount: 10,
+};
+
+export function createFuzzDescribe(optionsArg?: FuzzDescribeOptions): DescribeFuzz {
+    const options = { ...defaultOptions, ...optionsArg };
+    const testCountFromEnv = process.env.FUZZ_TEST_COUNT !== undefined ?
+        Number.parseInt(process.env.FUZZ_TEST_COUNT, 10) : undefined;
+    const testCount = testCountFromEnv ?? options.defaultTestCount;
+    const isStress = process.env.FUZZ_STRESS_RUN !== undefined && !!process.env.FUZZ_STRESS_RUN;
```

Comment:
You don't need the first condition here right? `!!undefined` is `false`

Reply:
linter complains without it, I figured might as well please the linter over disable

Reply:
ugh of course :(